style.css

```diff
@@ -814,6 +827,21 @@ span {
     display: none;
   }
 
+  .pro-btn {
+    background: #ff6b00;
+    font-family: inherit;
+    font-size: 17px;
+    font-weight: 700;
+    position: absolute;
+    bottom: 0;
+    right: 0;
+    left: 0;
+    height: 48px;
+    color: #fff;
+    width: 100%;
+    border: none;
+  }
```

Comment:
- [ ] Good job adding the see project button on hover effect as suggest by the previous reviewer [here](https://github.com/RitaDee/portfolio/pull/20#:~:text=Kindly%20make%20sure%20on%20the%20desktop%20version%20user%20can%20be%20able%20to%20see%20the%20See%20Project%20button%20maybe%20by%20hovering%20over%20the%20project%20card.%20At%20this%20moment%20we%20are%20not%20able%20to%20other%20rest%20five%20card) :clap: , however, while doing you have changed your design, please make it look back like the Figma design :record_button: 

Your current design:
![YourStacks](https://user-images.githubusercontent.com/97834160/211058717-a112e07b-0a12-4812-b151-0671296cd906.png)

Your previous & figma:
![figstacks](https://user-images.githubusercontent.com/97834160/211058797-55c7c9ed-3ad6-4c59-9474-91450a6e9116.png)